### DISCUSSION TITLE
Not able to load mantine css inside iframe (again)

### DISCUSSION THREAD

**Discussion by yuki-js** [mantinedev/mantine#5532] [Q&A]
I tried using Mantine with `react-frame-component` but CSS is not applied correctly.
A similar question was made before, but since Mantine v7 has become independent with Emotion, the answer is not working any more. 
I tried `getRootElement` like this: 
```jsx
 <MantineProvider getRootElement={() => ref.current.contentDocument.body}>{children}</MantineProvider>
```
and also tried injecting `style` element directly, but the CSS variable seems not applied.
![image](https://github.com/mantinedev/mantine/assets/20838151/3ab3c502-f009-4211-af75-080ba92498dd)
https://github.com/orgs/mantinedev/discussions/598

**Comment by nimatf790** [comment]
You have to specify the `data-mantine-color-scheme` property on your `html` tag, try this:
```
<Frame
  initialContent='<!DOCTYPE html><html  data-mantine-color-scheme="light"><head>...'
  mountTarget='#mountHere'
  >
</Frame>
```
